saltproc/serpent_depcode.py

```diff
@@ -175,20 +188,33 @@ def convert_nuclide_code_to_name(self, nuc_code):
         """
 
         if '.' in str(nuc_code):
-            nuc_code = pyname.zzzaaa_to_id(int(nuc_code.split('.')[0]))
+            nuc_code = int(nuc_code.split('.')[0])
+            if self.zaid_convention == 'serpent':
+                nuc_code = pyname.zzzaaa_to_id(nuc_code)
+            if self.zaid_convention in ('mcnp', 'nndc'):
```

Comment:
Consider changing to `elif` here, then you could add an `else` statement to make sure `self.zaid_convention` is an expected value.